Model/Page.php

```diff
@@ -134,6 +134,11 @@
     protected $target;
 
     /**
+     * @var int
+     */
+    protected $targetRedirectType = 302;
```

Comment:
Haven't got this working with `private` as the extending class won't have access to the field anymore and so doctrine can't access it too. This will probably affect all bundles which "depend" on the easy-extends bundle.